src/component/Popup/FeatureInfo/FeatureInfo.tsx

```diff
@@ -8,23 +8,91 @@ const isEqual = require('lodash/isEqual');
 import './FeatureInfo.less';
 
 interface DefaultFeatureInfoProps {
-  offset: [number, number];
+  /**
+   * The width of the popup.
+   * @type {Number}
+   */
+  width: number;
+
+  /**
+   * Offsets in pixels used when positioning the overlay. The first element
+   * in the array is the horizontal offset. A positive value shifts the
+   * overlay right. The second element in the array is the vertical offset.
+   * A positive value shifts the overlay down.
+   * @type {Array}
+   */
+  offset: number[];
+
+  /**
+   * Whether event propagation to the map viewport should be stopped. If true
+   * the overlay is placed in the same container as that of the controls
+   * (CSS class name ol-overlaycontainer-stopevent); if false it is placed in
+   * the container with CSS class name ol-overlaycontainer.
+   * @type {Boolean}
+   */
   stopEvent: boolean;
+
+  /**
+   * Whether the overlay is inserted first in the overlay container, or
+   * appended. If the overlay is placed in the same container as that of the
+   * controls (see the stopEvent option) you will probably set insertFirst to
+   * true so the overlay is displayed below the controls.
+   * @type {Boolean}
+   */
   insertFirst: boolean;
+
+  /**
+   * If set to true the map is panned when calling setPosition, so that the
+   * overlay is entirely visible in the current viewport.
+   * @type {Boolean}
+   */
   autoPan: boolean;
+
+  /**
+   * The animation options used to pan the overlay into view. This animation
+   * is only used when autoPan is enabled. A duration and easing may be
+   * provided to customize the animation.
+   * @type {Object}
+   */
   autoPanAnimationDuration: number;
+
   autoPanMargin: number;
+
   positioning: OverlayPositioning;
 }
 
 interface FeatureInfoProps extends Partial<DefaultFeatureInfoProps> {
 
   map: OlMap;
   position: number[];
+  /**
+   * Defines how the overlay is actually positioned with respect to its
+   * position property. Possible values are 'bottom-left', 'bottom-center',
+   * 'bottom-right', 'center-left', 'center-center', 'center-right',
+   * 'top-left', 'top-center', and 'top-right'.
+   * @type {String}
+   */
+  positioning: OverlayPositioning;
```

Comment:
dublicated prop definition (s. line 61 above)

Reply:
no, its another prop ("position") and both are used, see e.g. line 183

Reply:
But the name is not `position` but `positioning` here?? The same as already declared in line 61 above..

[Line 61](https://github.com/terrestris/react-geo-baseclient/blob/23761236a12bb65d683da806c59f586ec94ed266/src/component/Popup/FeatureInfo/FeatureInfo.tsx#L61) vs. [your change](https://github.com/terrestris/react-geo-baseclient/blob/23761236a12bb65d683da806c59f586ec94ed266/src/component/Popup/FeatureInfo/FeatureInfo.tsx#L75)